ibm_watson.gemspec

```diff
@@ -12,7 +12,7 @@ Gem::Specification.new do |spec|
   spec.summary = "Official client library to use the IBM Watson Services"
   spec.homepage = "https://www.github.com/watson-developer-cloud"
   spec.licenses = ["Apache-2.0"]
-  spec.required_ruby_version = ">= 2.3"
```

Comment:
In reference to my previous comment, maybe we leave this same and change it during next major release.

Reply:
also had to changes appveyor back to 2.3 for the build to pass.